src/components/pages/Home/HomeContainer.js

```diff
@@ -29,26 +30,26 @@ function HomeContainer({
         // if user is authenticated we can use the authService to snag some user info.
         // isSubscribed is a boolean toggle that we're using to clean up our useEffect.
         if (isSubscribed) {
-          setUserInfo(info);
-          getUsername(info.name);
+          getUserInfo(info);
         }
       })
       .catch(err => {
         isSubscribed = false;
-        return setUserInfo(null);
+        return getUserInfo({});
       });
     return () => (isSubscribed = false);
   }, [memoAuthService]);
 
   useEffect(() => {
     getUserTopics(getTopics);
```

Comment:
is `getTopics` coming from props or from the import?  Why both is it in both props and being imported?  I don't see where it is being placed on props 🤷‍♂️